components/esp_websocket_client/examples/target/generate_certs.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Generate CA, Server, and Client certificates automatically
+
+# Create directories if they don't exist
+mkdir -p main/certs/server
+
+echo "Generating CA certificate..."
+openssl genrsa -out main/certs/ca_key.pem 2048
+openssl req -new -x509 -days 3650 -key main/certs/ca_key.pem -out main/certs/ca_cert.pem -subj "/C=US/ST=State/L=City/O=Organization/CN=TestCA"
+
+echo "Generating Server certificate..."
+openssl genrsa -out main/certs/server/server_key.pem 2048
+openssl req -new -key main/certs/server/server_key.pem -out server_csr.pem -subj "/C=US/ST=State/L=City/O=Organization/CN=localhost"
```

Comment:
Shouldn't the `CN` match the name of the server in ESP32? I think `localhost' would evaluate to ESP32's address, correct?